packages/frontend/src/pages/scaling/summary/view/ScalingSummaryView.tsx

```diff
@@ -16,7 +16,11 @@ import {
 import { TableView } from '../../../../components/table/TableView'
 import { RowConfig } from '../../../../components/table/types'
 import { Tabs } from '../../../../components/Tabs'
-import { ScalingSummaryViewEntry } from '../types'
+import {
+  ScalingSummaryViewEntry,
+  ScalingSummaryViewEntryLayer2,
+  ScalingSummaryViewEntryLayer3,
+} from '../types'
 
 export interface ScalingSummaryViewProps {
   items: ScalingSummaryViewEntry[]
```

Comment:
maybe lets do it this way: 
```tsx
{
  layer2s: ScalingL2SummaryViewEntry[]
  layer3s: ScalingL3SummaryViewEntry[]
}
```

this way we don't have to have some weird function handling both cases and we don't have to filter to get l2s